tests/unit/src/main/scala/tests/TestingServer.scala

```diff
@@ -138,7 +138,7 @@ final class TestingServer(
     isReliableFileWatcher = Properties.isLinux
   )
   server.connectToLanguageClient(client)
-
+  val Docstring: Regex = " *\\/?\\*.*".r
```

Comment:
This file has a lot of unrelated changes, can you revert them? It will be easier to review them.